### PR TITLE
Added additional parsing for simple CUE-OUT with Duration key

### PR DESCRIPTION
**Pull request by ahardwick**
The following was not parsed by any of the existing CUE-OUT lines

#EXT-X-CUE-OUT:DURATION=120

I added a new parse rather than modifying the existing regex for _cueout_simple

### CODE REVIEW

File: tests/playlists.py [hunk 1]
```diff
@@ -776,6 +776,18 @@
 2.aac
 '''
 
+CUE_OUT_WITH_DURATION_KEY_PLAYLIST = '''#EXTM3U
+#EXT-X-TARGETDURATION:6
+#EXT-X-CUE-OUT:DURATION=11.52
```
Comment (leandromoreira) [inline tests/playlists.py:781]: I think there is also a case for `#EXT-X-CUE-OUT:30.000` https://docs.aws.amazon.com/mediatailor/latest/ug/hls-ad-markers.html#hls-ad-markers-cue

**Review by leandromoreira — CHANGES_REQUESTED** [review]

File: m3u8/parser.py [hunk 2]
```diff
@@ -375,6 +375,15 @@ def _cueout_envivio(line, state, prevline):
     else:
         return None
 
+def _cueout_duration(line):
```
Comment (davemevans) [inline m3u8/parser.py:378]: This doesn't seem to get called - you need to add it to `_parse_cueout`

File: tests/playlists.py [hunk 3]
```diff
@@ -776,6 +776,18 @@
 2.aac
 '''
 
+CUE_OUT_WITH_DURATION_KEY_PLAYLIST = '''#EXTM3U
```
Comment (davemevans) [inline tests/playlists.py:779]: This doesn't appear to get used in a test anywhere ...

File: tests/playlists.py (hunk 1 above)
Comment (davemevans) [reply tests/playlists.py:781]: That case should be handled by existing code (`_cueout_simple`), assuming this PR doesn't change that.

**Comment by ahardwick** [comment]
I'll updated "_parse_cueout()". Apologies, I meant to add that.

File: tests/playlists.py (hunk 1 above)
Comment (ahardwick) [reply tests/playlists.py:781]: It is handled by the existing cue-out simple

File: tests/playlists.py (hunk 3 above)
Comment (ahardwick) [reply tests/playlists.py:779]: Updated parser.py "_parse_cueout"

File: m3u8/parser.py (hunk 2 above)
Comment (ahardwick) [reply m3u8/parser.py:378]: updated parser.py "_parse_cueout"

File: tests/playlists.py (hunk 3 above)
Comment (ahardwick) [reply tests/playlists.py:779]: I didn't include a test as it didn't seem needed, similar to _cueout_simple.  I included it for reference on the notation.

**Review by leandromoreira — APPROVED** [review]

**Review by mauricioabreu — APPROVED** [review]
Good job and thank you for adding the test with duration.

**Pull request merged**